packages/testing/src/MockProviders.tsx

```diff
@@ -6,6 +6,8 @@ import React from 'react'
 
 import { AuthProvider } from '@redwoodjs/auth'
 import type { AuthContextInterface } from '@redwoodjs/auth'
+// @ts-ignore
```

Comment:
@peterp I'm not sure what to do here. We use `// @ts-ignore` in MockRouter.tsx, why can't I do that here as well?

Reply:
Maybe try `// @ts-expect-error - <reason why we expect an error>`

Reply:
Thanks!